wagtail/wagtailusers/templates/wagtailusers/users/create.html

```diff
@@ -24,13 +24,15 @@
                     {% include "wagtailadmin/shared/field_as_li.html" with field=form.last_name %}
                     {% include "wagtailadmin/shared/field_as_li.html" with field=form.password1 %}
                     {% include "wagtailadmin/shared/field_as_li.html" with field=form.password2 %}
-                    
+
                     <li><a href="#roles" class="button lowpriority tab-toggle icon icon-arrow-right-after" />{% trans "Roles" %}</a></li>
                 </ul>
             </section>
             <section id="roles" class="nice-padding">
                 <ul class="fields">
-                    {% include "wagtailadmin/shared/field_as_li.html" with field=form.is_superuser %}
+                    {% if request.user.is_superuser %}
```

Comment:
I'd like to avoid having an explicit `request.user.is_superuser` check here, if possible - business rules should be kept out of templates (admittedly, this is a fairly trivial one...) and it duplicates the logic in UserCreationForm / UserEditForm. My preferred approach would be:

* UserCreationForm / UserEditForm check user.is_superuser within the `__init__` method, and dynamically add the is_superuser checkbox to the form if true
* the templates then check for the presence of `is_superuser` on the form, and skip it if not present

This hopefully eliminates the need for a dedicated `clean_is_superuser` method, too.